### PR TITLE
Updated version number in Exome pipeine overview

### PR DESCRIPTION
**Pull request by ekiernan**

### CODE REVIEW

**Review by jessicaway — APPROVED** [review]

**Comment by jessicaway** [comment]
The validation error is for the GCD somatic pipeline. This is because of the merge we did last week that added changes we had already pushed to staging. We can ignore the error for now. It will stop happening once we merge develop to staging

**Review by nikellepetrillo — APPROVED** [review]

**Comment by ekiernan** [comment]
Thanks @jessicaway . I can't merge this because I don't have admin. Are you able to merge?

**Pull request merged**
